src/components/ProductsFilter/CategoriesFilter/CategoriesFilter.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+export function CategoriesFilter() {
+  return (
+    <li className='flex-column products-filter-type'>
+      <h3 className='filter-heading'>Categories</h3>
+      <div className='form-check'>
+        <input type='checkbox' value='racket' id='rackets' />
+        <label className='form-label-inline' htmlFor='rackets'>
+          Rackets
+        </label>
+      </div>
+
+      <div className='form-check'>
+        <input type='checkbox' value='shuttle' id='shuttles' />
+        <label className='form-label-inline' htmlFor='shuttles'>
+          Shuttles
+        </label>
+      </div>
+
+      <div className='form-check'>
+        <input type='checkbox' value='bag' id='badminton-bags' />
+        <label className='form-label-inline' htmlFor='badminton-bags'>
+          Badminton Bags
+        </label>
+      </div>
+
+      <div className='form-check'>
```

Comment:
Just a suggestion. `form-check` div is repeating in this file and in `FilterByAvailabity.jsx`. See if you can make it dynamic. It will be easier to add more categories as well.

Reply:
Thanks for suggesting. I will work on it.